app/assets/stylesheets/components.scss

```diff
@@ -410,6 +410,8 @@ span.showHide-open-all {
     padding-left: 0;
     span {
       display: block;
+      -ms-word-break: break-all;
```

Comment:
What are your thoughts on using something like https://github.com/ai/autoprefixer-rails?

Reply:
Looks interesting - not something I've come across before, but it seems less dirty than littering up the css directly. I'll look into it.

Reply:
@andrewgarner Have asked around and it's used elsewhere in gds, so will look at implementing that if needed in future. In the short term, though, I've refactored based on Vanita's comments.

Reply:
👍